import-export-cli/integration/throttlePolicy_test.go

```diff
@@ -0,0 +1,454 @@
+				CtlUser:  testutils.Credentials{Username: user.CtlUser.Username, Password: user.CtlUser.Password},
+				Policy:   throttlePolicy,
+				Type:     applicationThrottlePolicyFlag,
+				DestAPIM: prod,
+				Update:   false,
+			}
+			adminUsername := adminservices.AdminUsername
+			adminPassword := adminservices.AdminPassword
+			if isTenantUser(args.CtlUser.Username, TENANT1) {
+				adminUsername = adminUsername + "@" + TENANT1
+			}
+			testutils.ValidateThrottlePolicyImportUpdateConflict(t, args, adminUsername, adminPassword, apim.ApplicationThrottlePolicyType)
+		})
+	}
+}
+
+// Import an already existing Advanced Throttling Policy to the destination env without update
+func TestImportUpdateConflictAdvancedThrottlePolicy(t *testing.T) {
+
+	for _, user := range testCaseUsers {
+		t.Run(user.Description, func(t *testing.T) {
+			prod := GetProdClient()
+
+			newPolicy := testutils.AddNewThrottlePolicy(t, prod, user.Admin.Username, user.Admin.Password, apim.AdvancedThrottlePolicyType)
+			throttlePolicy, _ := testutils.PolicyStructToMap(newPolicy)
+			args := &testutils.PolicyImportExportTestArgs{
+				CtlUser:  testutils.Credentials{Username: user.CtlUser.Username, Password: user.CtlUser.Password},
+				Policy:   throttlePolicy,
+				Type:     advancedThrottlePolicyFlag,
+				DestAPIM: prod,
+				Update:   false,
+			}
+			adminUsername := adminservices.AdminUsername
+			adminPassword := adminservices.AdminPassword
+			if isTenantUser(args.CtlUser.Username, TENANT1) {
+				adminUsername = adminUsername + "@" + TENANT1
+			}
+			testutils.ValidateThrottlePolicyImportUpdateConflict(t, args, adminUsername, adminPassword, apim.AdvancedThrottlePolicyType)
+		})
+	}
+}
+
+// Import an already existing Custom Throttling Policy to the destination env without update
+func TestImportUpdateConflictCustomThrottlePolicy(t *testing.T) {
+	for _, user := range testCaseUsers {
+		if isTenantUser(user.CtlUser.Username, TENANT1) {
+			continue
+		}
+		t.Run(user.Description, func(t *testing.T) {
+			prod := GetProdClient()
+
+			newPolicy := testutils.AddNewThrottlePolicy(t, prod, user.Admin.Username, user.Admin.Password, apim.CustomThrottlePolicyType)
+			throttlePolicy, _ := testutils.PolicyStructToMap(newPolicy)
+			args := &testutils.PolicyImportExportTestArgs{
+				CtlUser:  testutils.Credentials{Username: user.CtlUser.Username, Password: user.CtlUser.Password},
+				Policy:   throttlePolicy,
+				Type:     customThrottlePolicyFlag,
+				DestAPIM: prod,
+				Update:   false,
+			}
+			adminUsername := adminservices.AdminUsername
+			adminPassword := adminservices.AdminPassword
+			testutils.ValidateThrottlePolicyImportUpdateConflict(t, args, adminUsername, adminPassword, apim.CustomThrottlePolicyType)
+		})
+	}
+}
+
+// Get  Throttle Policy List APICTL output and check whether all policies are included
+func TestGetThrottlePoliciesList(t *testing.T) {
+	for _, user := range testCaseUsers {
+		if isTenantUser(user.CtlUser.Username, TENANT1) {
+			continue
```

Comment:
Do we need to skip tenant users when retrieving the policies as well? 
If this limitation is only for custom policies, let's mention it as a comment inside this test, but the test should run for all the 4 users. Otherwise, that is unfair for other policy types.

Reply:
In TestGetThrottlePoliciesListWithJsonPrettyFormat also we are running for all the 4 users.

Reply:
fixed with 479b4168267adb70bde906548a563dade6a07df8